hi-ml-multimodal/src/health_multimodal/image/model/model.py

```diff
@@ -112,7 +125,7 @@ def train(self, mode: bool = True) -> Any:
             self.projector.train(mode=False)
         return self
 
-    def forward(self, x: torch.Tensor) -> ImageModelOutput:
+    def forward(self, x: torch.Tensor, *args: Any, **kwargs: Any) -> ImageModelOutput:  # type: ignore[override]
```

Comment:
Why are we allowing and then ignoring potential args and kwargs here? If someone passes in two tensors to this function, there's no indication that this is not supported - it's very possible this fails silently completely unknown to the user.

We are designing this code to allow users to use classes that inherit from `BaseImageModel` interchangeably, so there's a reasonable chance that a user swaps out one class with another and the model forward pass is run incorrectly but silently.

Reply:
I agree. I thought we were ignoring mypy errors for the base class, not for this.

Reply:
I've removed the unused args/kwargs now. The mypy issue occurs in the child class:
`model.py:128: error: Signature of "forward" incompatible with supertype "BaseImageModel"` 
... so I think we need to ignore it here. This might be a bit annoying as I guess all children of `BaseImageModel` will need to do something similar to appease mypy.